config/locales/en.yml

```diff
@@ -2835,7 +2835,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   tag_rules: "Tag Rules"
   shop_preferences: "Shop Preferences"
   enterprise_fee_whole_order: Whole order
-  enterprise_fee_by: "%{type} fee by %{role} %{enterprise_name}"
+  enterprise_fee_by: "%{name} fee by %{role} %{enterprise_name}"
```

Comment:
I wonder if this can cause problems. If other languages are not updated then the old variable is required but won't be set. The solution would be to introduce a new translation key. But let's monitor this in the release and see what Transifex does. Maybe it updates this automatically which would be great.

Reply:
@mkllnk so, this cause issue: https://app.bugsnag.com/yaycode/openfoodnetwork-uk/errors/633477967f4d38000830e14e?event_id=6334779600979f7f70b60000&i=sk&m=nw

I'll update the PR by creating a new key.

Reply:
I've introduced a new key `enterprise_fee_by_name`: 5e1d7691d2

Moving to `Test Ready`, I assume this is small enough to not be reviewed.